src/game/server/gamecontroller.h

```diff
@@ -3,6 +3,7 @@
 #ifndef GAME_SERVER_GAMECONTROLLER_H
 #define GAME_SERVER_GAMECONTROLLER_H
 
+#include <cstdio>
```

Comment:
What is this include for?

Reply:
Oh that's not meant to be there, leftover from debugging

```suggestion
```